addon/components/polaris-popover/content.js

```diff
@@ -82,29 +81,40 @@ export default Component.extend({
       return;
     }
 
-    const trigger = Ember$(
+    const trigger = document.querySelector(
```

Comment:
This will blow up if ever anybody tries using this addon in fastboot, where `document` is unavailable. Having said that, we probably have a bunch of stuff which is fastboot-unfriendly (anything using the `didInsertElement ` hook, for example, or JQuery (which maybe you've got rid of completely in this PR? 🙏)) so maybe let's not worry about it at this point and update for fastboot compatibility if we ever need it in the future?